src/main/java/duke/Event.java

```diff
@@ -0,0 +1,15 @@
+package duke;
+
+public class Event extends Task {
+    protected String timing;
+
+    public Event(String description, String location) {
+        super(description);
+        this.timing = timing;
```

Comment:
Perhaps timing should be date so that it is more consistent with the deadline task?